CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 
+* [ENHANCEMENT] Added hedged request metric `backend_hedged_roundtrips_total` and general purpose `backend_request_duration_seconds` metric that will supersede soon to be deprecated storage specific metrics (eg `request_duration_seconds`). [#790](https://github.com/grafana/tempo/pull/790) (@JosephWoodward)
```

Comment:
Can you list the deprecated 3 explicitly? 

Also all the metrics are prefixed by `tempodb_`

Reply:
> Can you list the deprecated 3 explicitly?
Yeah, no problem

> Also all the metrics are prefixed by `tempodb_`
Good catch! I forgot about the namespace. That's done now.